backend/core/src/auth/constants.ts

```diff
@@ -10,18 +10,18 @@ export const AUTH_COOKIE_OPTIONS: CookieOptions = {
   httpOnly: true,
   // since our local env doesn't have an https cert we can't be secure. Hosted envs should be secure
   secure: process.env.NODE_ENV !== "development",
-  sameSite: true,
+  sameSite: process.env.NODE_ENV === "development" ? "strict" : "none",
```

Comment:
Reading the link you referenced, it sounds like "none" should work for our deployed sites. Is the reason it still has to be done for development is because "secure" is false?

Reply:
yeah, since locally we aren't working with an https cert `secure` needs to be false, but that breaks `sameSite` of none